src/history/repeat_commands/repeat_commands_generic.ts

```diff
@@ -0,0 +1,39 @@
+import { deepCopy } from "../../helpers";
+import { Command, Getters } from "../../types";
+
+export const genericTransforms = [
+  transformSheetDependantCommand,
+  transformTargetDependantCommand,
+  transformPositionDependantCommand,
+];
+
+export function transformSheetDependantCommand<T extends Command>(getters: Getters, command: T): T {
+  if (!("sheetId" in command)) return command;
+
+  return { ...deepCopy(command), sheetId: getters.getActiveSheetId() };
+}
+
+export function transformTargetDependantCommand<T extends Command>(
+  getters: Getters,
+  command: T
+): T {
+  if (!("target" in command) || !Array.isArray(command.target)) return command;
```

Comment:
Why is it needed to be an Array ? To ensure it's a range ? 
And shouldn't we check that the target isn't a fixed col/row zone ?

Reply:
I need to check that it's an array because of the `PasteCFCommand`, which have a target that is a cell... I should probably just change this command, it's not coherent to have command whose target isn't like all the others.

Target are an array of `Zone`, and there is no concept of fixed col/row in `Zone`.

Reply:
Added a commit to rename the payload of `PasteCFCommand`. No miragtion needed since it's a UI command.